extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -68,6 +69,53 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
     this.storeService_.subscribe(StateProperty.RTL_STATE, (rtlState) => {
       this.onRtlStateUpdate_(rtlState);
     });
+
+    this.storeService_.subscribe(StateProperty.PAGE_SIZE, (pageSizeState) => {
+      this.shouldTagFlip_(pageSizeState);
```

Comment:
Good point, however I tried this and since it was in a measure element, it caused a bit of problems with the DOM syncing up properly as I would be returning a promise, not a boolean. I think it would be better to just rename the method to something like `flipTagIfOffscreen_`.